packages/make-styles-webpack-loader/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui/make-styles-webpack-loader",
-  "version": "9.0.0",
+  "version": "9.0.0-alpha.0",
```

Comment:
Unfortunately yes 😢

---

It's happened in 8cfb390990af78782dcc7da09aea00f8d2ca05bc:

![image](https://user-images.githubusercontent.com/14183168/126167783-4f7c83fa-db16-4b31-be23-06149b8abbef.png)

(due missing config for `beachball`, this PR also adds it)

Reply:
when did it happen ? (oh 3 days ago... we are screwed :D ) there is some time period when package can be deleted from registry

Reply:
Yeah, otherwise we can receive an interesting side-effect once we will decide to release `9.0.0` on other packages:

Edit 👇

```
npm info @fluentui/make-styles-webpack-loader
npm ERR! code E404
npm ERR! 404 Unpublished by layershifter on 2021-07-19T13:36:31.697Z
```

Reply:
well looks like our 1st stable major vNext is gonna be `9.0.1` 

- we should create issue for that so everyone is aware

Reply:
I created an internal thread & created a Github issue #19000.

Reply:
We could still `npm deprecate` 9.0.0, so at least it won't be installed automatically.